typescript/05/main.ts

```diff
@@ -0,0 +1,45 @@
+type Subjects = 'math' | 'geo' | 'english';
```

Comment:
Co jeśli pojawi się nowy przedmiot? Za każdym razem będziesz go dodawał? Raczej ograniczenie tego zbioru uznałbym za bardziej problematyczne niż pomocne. Chyba, że jest jakiś ministerialny zbór wszystkich dozwolonych przedmiotów :)

Reply:
Masz rację, usunąłem ten typ. Chociaż myślę, że posiadanie listy przedmiotow jest interesującą rzeczą z racji na mniejszą ilość przypadkowych stringów

Reply:
Zdecydowanie tak, ale czy będziesz ją kontrolował za każdym razem jak pojawi się nowy rok szkolny? Alb jeśli ktoś napisze język polski albo Język Polski, a może Język polski? To byłoby bardzo irytujące, gdy użytkownicy dzwonili do Ciebie z taką pierdołą - bo nie działa :D